vector/src/main/java/im/vector/app/VectorApplication.kt

```diff
@@ -140,6 +144,11 @@ class VectorApplication :
         Mavericks.initialize(debugMode = false)
         EpoxyController.defaultDiffingHandler = EpoxyAsyncUtil.getAsyncBackgroundHandler()
         EpoxyController.defaultModelBuildingHandler = EpoxyAsyncUtil.getAsyncBackgroundHandler()
+        Carousel.setDefaultGlobalSnapHelperFactory(object : Carousel.SnapHelperFactory() {
```

Comment:
Maybe we should extract this into a dedicated private method (e.g. `initDefaultSnapHelper`) to reduce the size of the `onCreate` method and ease the readability.

Reply:
I've moved it to `configureEpoxy` method along with other epoxy related things